notebooks/binary_predicates.ipynb

```diff
@@ -0,0 +1,1157 @@
+{
```

Comment:
<blockquote>There are three relationships that a Point can have with a LineString: Disjoint, Point is equal to one of the LineString boundary points, and point falls between the LineString boundary points. </blockquote><p>Is there a name for the latter 2 relationships?</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cuspatial/pull/1156/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>